code/modules/mob/living/say.dm

```diff
@@ -259,20 +259,29 @@ var/list/department_radio_keys = list(
 	return 0
 
 /mob/living/send_speech(var/datum/speech/speech, var/message_range=7, var/bubble_type) // what is bubble type?
+	var/visual_range = message_range //the range of hearers who can see that something was said, but not hear the message
 	say_testing(src, "/mob/living/send_speech() start, msg = [speech.message]; message_range = [message_range]; language = [speech.language ? speech.language.name : "None"]; speaker = [speech.speaker];")
 	if(isnull(message_range))
 		message_range = 7
 
-	var/list/listeners = get_hearers_in_view(message_range, speech.speaker) | observers
+	message_range = atmospheric_speech(speech,message_range)
+
+	var/list/total_listeners = get_hearers_in_view(visual_range, speech.speaker)
```

Comment:
side note when are we going to add astar but for voices so you can hear people off screen.

Reply:
N3X15 is working on some fancy positional audio stuff for SS13, which I imagine might include things like that.

Reply:
I was kidding around.  That'd be sick.